plugins/acl/project-role-based/src/main/java/org/apache/cloudstack/acl/ProjectRoleBasedApiAccessChecker.java

```diff
@@ -103,6 +103,11 @@ public boolean checkAccess(User user, String apiCommandName) throws PermissionDe
         throw new UnavailableCommandException("The API " + apiCommandName + " does not exist or is not available for this account/user in project "+project.getUuid());
     }
 
+    @Override
+    public boolean checkAccess(Account account, String apiCommandName) throws PermissionDeniedException {
+        return true;
```

Comment:
Could this cause any issue, cc @Pearl1594 ?

Reply:
This doesn't seem to affect the project roles feature.

Reply:
thanks for confirming this @Pearl1594